src/components/app/details/triggerView/workflow/nodes/triggerCINode.tsx

```diff
@@ -48,15 +48,17 @@ export class TriggerCINode extends Component<TriggerCINodeProps> {
         this.props.history.push(this.getCIDetailsURL())
     }
 
+    hideDetails(status: string = '') {
+      return status === DEFAULT_STATUS.toLowerCase() || status === 'not triggered' || status === 'not deployed' || status ===''
```

Comment:
not triggered &  not deployed can extract outside

Reply:
done